source/conf.py

```diff
@@ -82,6 +82,7 @@
 .. _oneAPI:  https://oneapi.com
 .. _SYCL: https://www.khronos.org/sycl/
 .. |SYCL_SPEC| replace:: `SYCL Specification`_
+.. |SYCL_SPEC_HEADER_FILES| replace:: SYCL Specification `Section 4.3 <https://registry.khronos.org/SYCL/specs/sycl-2020-provisional.pdf#page=56&zoom=100,96,189>`__
```

Comment:
Links should be to the final specification, and not to the provisional (draft) version.

The place to find links is https://registry.khronos.org/SYCL/

So I think this reference should be to https://registry.khronos.org/SYCL/specs/sycl-2020/pdf/sycl-2020.pdf

Also, it might make more sense to link to the HTML version of the specification, instead of the PDF version, since those following the link will already be in a web browser.

Reply:
Changed to HTML version with last commit amend.